apps/beets-frontend-v3/lib/modules/reliquary/ReliquaryProvider.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
This looks dangerous. Cannot we fix the root cause?

Reply:
fixed!